config/config.go

```diff
@@ -722,6 +722,14 @@ func (self *CGRConfig) checkConfigSanity() error {
 				utils.DispatcherS, self.dispatcherSCfg.DispatchingStrategy)
 		}
 	}
+	// Scheduler
+	if self.schedulerCfg != nil && self.schedulerCfg.Enabled {
```

Comment:
please remove the need of schedulerCfg.Enabled here since we could be setting up schedCDRsConns even if scheduler is disabled